hack/test-canary.sh

```diff
@@ -95,13 +96,34 @@ function run() {
 
 # Helper function to clean up log and json files.
 function clean() {
-    kubectl get pods --all-namespaces | grep ccm | awk '{print $1}' | xargs kubectl delete ns
-    rm "${TEST_DIR}/${TEST_PREFIX}*"
+    echo "ensuring fresh \$START."
```

Comment:
In monitor mode we no longer run once and exist - so we need a function to clean up resources between each iteration.